client/src/components/pages/HomePage.js

```diff
@@ -1,21 +1,43 @@
-import React from "react";
+import React, { useState } from "react";
 import Input from "../Input/Input";
 import Header from "../Header/Header";
-import styled from "styled-components/macro";
+import Container from "../Container";
+import { useHistory } from "react-router-dom";
+//import { setName } from "../../utils/Storage";
 
-const Container = styled.div`
-  display: flex;
-  align-items: center;
-  justify-content: center;
-`;
+function HomePage() {
+  const history = useHistory();
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const newName = [name];
+    setName("");
+    localStorage.setItem("name", JSON.stringify(newName));
+    history.push("/startpage");
+  };
```

Comment:
Maybe I'm just noobing around, why does this have to be async, because of the localStorage 😅 ?

`async function(e){...}`

...but at one point I would outsource the localStorage function...

Reply:
You are right, it doesn´t have to be async.